variables.tf

```diff
@@ -165,6 +165,13 @@ variable "remote_metastores" {
   default     = []
 }
 
+#list of maps, example: [ {endpoint="vpce1", port="9083", prefix="pre1", writable-whitelist="db1,test", vpc_id = "vpc-123456", subnets = "subnet1,subnet2", security_group_id="sg1"  } ]
+variable "remote_region_metastores" {
+  description = "List of VPC endpoint services to federate Metastores in other region,other accounts."
```

Comment:
Again, should explicitly mention Alluxio here.

Or maybe not, if `remote_region_metastores` is not always used with Alluxio.